crates/re_types_builder/src/codegen/python.rs

```diff
@@ -1659,6 +1665,13 @@ fn quote_init_method(obj: &Object, ext_class: &ExtensionClass, objects: &Objects
             .iter()
             .filter_map(|field| {
                 if field.docs.doc.is_empty() {
+                    if !field.is_testing() && obj.fields.len() > 1 {
+                        reporter.error(
+                            &field.virtpath,
+                            &field.fqname,
+                            format!("Field {} is missing documentation", field.name),
```

Comment:
would be nice to have it not in python but in a `common` location. But whatever, this will do